monitors.yml

```diff
@@ -195,3 +195,18 @@ monitors:
         warning: 400000
     query: avg:zuul.pipeline.check_github.job.bonnyci_run_check.wait_time.max{host:zuul} > 800000
     type: metric alert
+  - message: |
+      {{#is_warning}}Only two nodes available{{/is_warning}} 
+      {{#is_alert}}No nodes available!{{/is_alert}} 
+
+      @slack-bonny-dd-alerts
+    multi: false
+    name: Available Nodes
+    options:
+      notify_no_data: false
+      require_full_window: false
+      thresholds:
+        critical: 0
```

Comment:
I actually think you want this to be 1? Since the query is using '<' operator, these thresholds are dropped in place, is my understanding. That would mean we'd only get critical if we had less than zero nodes ready, which will never happen.